doc/usage_guide.md

```diff
@@ -233,10 +233,25 @@ Note this example is for Linux and macOS usage.
 You can set environment variables for your connection string through the following steps:
 
 
+    # (linux/bash)
     # set environment variable MSSQL_SCRIPTER_CONNECTION_STRING with a connection string.
     $ export MSSQL_SCRIPTER_CONNECTION_STRING='Server=myserver;Database=mydb;User Id=myuser;Password=mypassword;'
     $ mssql-scripter
 
+    # (linux/bash)
     # set environment variable MSSQL_SCRIPTER_PASSWORD so no password input is required.
     $ export MSSQL_SCRIPTER_PASSWORD='[PLACEHOLDER]'
     $ mssql-scripter -S localhost -d AdventureWorks -U sa
+
+    # (windows)
+    # set environment variable MSSQL_SCRIPTER_CONNECTION_STRING with a connection string.
+    setx MSSQL_SCRIPTER_CONNECTION_STRING 'Server=myserver;Database=mydb;User Id=myuser;Password=mypassword;'
```

Comment:
Setting a system variable for this doesn't seem like the best thing to suggest - I think just using just `$ENV` here would be preferable (and then you also don't need to restart the environment either)

`$ENV:MSSQL_SCRIPTER_CONNECTION_STRING='Server=myserver;Database=mydb;User Id=myuser;Password=mypassword;'`